rasa_nlu_examples/classifiers/sparse_logistic_regression_intent_classifier.py

```diff
@@ -0,0 +1,46 @@
+import typing
+from typing import Any
+
+from rasa_nlu_examples.classifiers.sparse_sklearn_intent_classifier import (
+    SparseSklearnIntentClassifier,
+)
+
+if typing.TYPE_CHECKING:
+    import sklearn.linear_model
+
+
+class SparseLogisticRegressionIntentClassifier(SparseSklearnIntentClassifier):
+    r"""A logistic regression classifier using the sklearn framework with sparse features."""
+
+    defaults = {
+        # The following parameters and defaults are the same as the ones used by the
+        # current scikit-learn version (0.24.2). For some nice explanations on what
+        # these parameters and their defaults so, have a look at the scikit-learn docs:
+        # https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html
+        "C": 1.0,
+        "class_weight": None,
```

Comment:
For our use-case, wouldn't it make sense to have "balanced" as the base behavior?

Reply:
Hm, first of all, because I found it strange if the defaults are different from the ones I'm used to from sklearn. But if "in our use case" means that we usually have way too many examples for classes that we care less about ("hello") and few for the really interesting ones - I guess yes, we should make balanced default ^^.

Reply:
Changed to balanced as default.